index.html

```diff
@@ -0,0 +1,353 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Form Factors Explained</title>
+  <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
+  <script class="remove">
+   // All config options at https://respec.org/docs/ 
+   var respecConfig = {
+      subtitle: "Foldable devices and multi-monitor setups",
+      specStatus: "unofficial",
+      editors: [
+        { name: "Second Screen Community Group", url: "https://www.w3.org/community/webscreens/" },
+      ],
+      github: "webscreens/form-factors",
+      shortName: "form-factors",
+      xref: "web-platform",
+      group: "webscreens",
+      format: "markdown"
+    };
+  </script>
+</head>
+<body>
+
+<section id="abstract">
+## Abstract
+
+This explainer discusses new device [=form factors=] that are emerging and related terminology that is being established. This document proposes one naming scheme to be able to reason about and use these terms and concepts consistently in [related web specifications](#related-web-specifications). The use of these terms in other contexts may differ.
+</section>
+
+<!-- How to edit this file: https://respec.org/docs/ -->
+
+## Terminology and conceptual model
+
+><dfn data-export="">Form factor</dfn> is a hardware design aspect that defines and prescribes the size, shape, and other physical specifications of components, particularly in electronics. A form factor may represent a broad class of similarly sized components, or it may prescribe a specific standard. It may also define an entire system, as in a computer form factor. ([Wikipedia](https://en.wikipedia.org/wiki/Form_factor_(design)))
+
+![](images/relationship.svg)
```

Comment:
Perhaps "This explainer" should be "Multi-screen form factors" or similar?

Reply:
interesting thought. can you elaborate a bit more?

Reply:
1) It might help to clarify the "This explainer" node.
2) Defining terminology for a "traditional single-screen device" form factor (or similar) could help us more clearly convey the basis for existing paradigms (like window.screen) and how new form factors differ from that base state. Perhaps this definition could refer to the CSSOM View Draft's ["2.3. Web-exposed screen information"](https://drafts.csswg.org/cssom-view/#web-exposed-screen-information), which describes "the screen of the output device". MDN also [says](https://developer.mozilla.org/en-US/docs/Web/API/Screen) "The Screen interface represents a screen, usually the one on which the current window is being rendered, and is obtained using window.screen... Note that browsers determine which screen to report as current by detecting which screen has the center of the browser window." In traditional computing environments, this form factor was obvious or implicit (a desktop's single monitor, a laptop's single internal display panel, a phone or tablet's single screen). Making explicit terminology for that traditional form factor could help us more clearly define the distinguishing characteristics of multi-monitor setups, foldable devices, and new OS/WM composite/virtual screen paradigms.

Reply:
Addressed by 643691c.